Bottom: c74f2156bf1e9d5ddff05afc138cb73ffbdf5a3b
Top:    5608b28f262ba99d4ef9c543871d2ebb66e36144
Author: Ian Kent <raven@themaw.net>
Date:   2013-07-24 16:26:54 +0800

Refresh of autofs-5.0.7-unbundle-NOTSTRIP-from-DEBUG-so-they-dont-depend-on-each-other.patch

---

diff --git a/CHANGELOG b/CHANGELOG
index 897a2b4..42ea627 100644
--- a/CHANGELOG
+++ b/CHANGELOG
@@ -64,6 +64,7 @@
 - use compiler determined by configure instead of hard-coded ones.
 - remove hard-coded STRIP variable.
 - use LIBS for link libraries.
+- unbundle NOTSTRIP from DEBUG so they dont depend on each other.
 
 25/07/2012 autofs-5.0.7
 =======================
